src/frontend/packages/lib_video/package.json

```diff
@@ -84,11 +84,14 @@
     "zustand": "*"
   },
   "dependencies": {
+    "@streamroot/videojs-hlsjs-plugin": "1.0.16",
     "altamoon-robust-websocket": "1.0.3",
     "m3u8-parser": "6.2.0",
     "lib-common": "*",
     "lib-components": "*",
     "linkifyjs": "4.1.1",
+    "p2p-media-loader-hlsjs": "^0.6.2",
+    "process": "^0.11.10",
     "react-icalendar-link": "3.0.2",
```

Comment:
You should pin the version no?

```suggestion
    "p2p-media-loader-hlsjs": "0.6.2",
    "process": "0.11.10",
```

Reply:
Yes but don't remove `react-icalendar-link` please 😅